cmd/system/node.go

```diff
@@ -0,0 +1,138 @@
+package system
+
+import (
+	"fmt"
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	cp "github.com/otiai10/copy"
+	"github.com/spf13/cobra"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func getLatestVersion(version, channel string) (*string, error) {
+	resp, err := http.Get(fmt.Sprintf("https://nodejs.org/download/%s/%s", channel, version))
+	if err != nil {
+		return nil, err
+	}
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	regex := regexp.MustCompile(`(?m)node-(.*)-linux-.*"`)
+	result := regex.FindStringSubmatch(string(body))
+	if len(result) < 2 {
+		return nil, fmt.Errorf("could not find latest version for %s", version)
+	}
+	return &result[1], nil
+}
+
+func MakeInstallNode() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "node",
+		Short: "Install Node.js",
+		Long:  `Install the Node.js JavaScript runtime ✨🐢🚀✨`,
```

Comment:
The emojis are cute, but can you keep this consistent with the other apps?

The official homepage says: `"Node.js is a JavaScript runtime built on [Chrome's V8 JavaScript engine](https://v8.dev/).`"

Reply:
Just c&p the text from their github description! 😅 

I removed it